rfcs/0021-ckb-address-format/0021-ckb-address-format.md

```diff
@@ -38,13 +38,28 @@ payload = 0x01 | code_hash_index | single_arg
 To translate payload to lock script, one can convert code_hash_index to code_hash and hash_type with the following *popular code_hash table*. And single_arg as the args.
 
 
-| code_hash_index |        code_hash     |   code_type  |      args     |
-|:---------------:|----------------------|:------------:|---------------|
-|      0x00       | SECP256K1 + blake160 |     Type     |  blake160(PK) |
-|      0x01       | SECP256K1 + hash160  |     Type     |  hash160(PK)  |
+| code_hash_index |        code_hash     |   code_type  |          args           |
+|:---------------:|----------------------|:------------:|-------------------------|
+|      0x00       | SECP256K1 + blake160 |     Type     |  blake160(PK)*          |
+|      0x01       | SECP256K1 + multisig |     Type     |  multisig script hash** |
 
 \* The blake160 here means the leading 20 bytes truncation of Blake2b hash result.
-\* The hash160 here means Bitcoin address calculation *double hash* algorithm, which means `hash160(PK) := Ripemd160(SHA256(PK))`.
+
+\*\* The *multisig script hash* is the 20 bytes blake160 hash of multisig script. The multisig script should be assembled in the following format:
+
+```
+S | R | M | N | Pubkey1 | Pubkey2 | ...
+```
+
+Where S/R/M/N are four single byte unsigned integers, ranging from 0 to 255, and Pubkeys are SECP256K1 public keys. S is format version, currently fixed to 0. M/N means the user must provide M of N signatures to unlock the cell. And R means the provided signatures at least match the first R items of the Pubkey list.
```

Comment:
There are two kind of pub keys, compressed and uncompressed. It's not clear which one is used here.

Reply:
I believe it should be uncompressed version, right?

Reply:
Can you drop a hint here? @jjyr

Reply:
Compressed

In the code:

```
PUBKEY_SIZE 33
```

Double check with @jjyr

Reply:
see https://github.com/nervosnetwork/rfcs/pull/141#pullrequestreview-309693715